### PR TITLE
Issue #652 Printing messages for addons enable and disable

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

Addresses issue #652

### CODE REVIEW

File: cmd/minishift/cmd/addon/addon_disable.go [hunk 1]
```diff
@@ -44,16 +44,18 @@ func runDisableAddon(cmd *cobra.Command, args []string) {
 		atexit.ExitWithMessage(1, emptyDisableError)
 	}
 
-	addonName := args[0]
+	addOnName := args[0]
```
Comment (budhrg) [inline cmd/minishift/cmd/addon/addon_disable.go]: I don't think this name change is required. `Addon` is full word.

File: cmd/minishift/cmd/addon/addon_disable.go [hunk 2]
```diff
@@ -44,16 +44,18 @@ func runDisableAddon(cmd *cobra.Command, args []string) {
...
+		atexit.ExitWithMessage(0, fmt.Sprintf(noAddOnToDisableMessage, addOnName))
 	}
 
-	addOnConfig, err := addOnManager.Disable(addonName)
+	addOnConfig, err := addOnManager.Disable(addOnName)
 	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Unable to disable add-on %s: %s", addonName, err.Error()))
+		atexit.ExitWithMessage(1, fmt.Sprintf("Unable to disable add-on %s: %s", addOnName, err.Error()))
+	} else {
+		fmt.Println(fmt.Sprintf("Addon %s disabled", addOnName))
```
Comment (budhrg) [inline cmd/minishift/cmd/addon/addon_disable.go]: `Addon '%s' disabled` wrapping in single quote give visibility to name of addon

File: cmd/minishift/cmd/addon/addon_enable.go [hunk 3]
```diff
@@ -61,10 +61,12 @@ func runEnableAddon(cmd *cobra.Command, args []string) {
...
 
-func enableAddon(addOnManager *manager.AddOnManager, addonName string, priority int) {
-	addOnConfig, err := addOnManager.Enable(addonName, priority)
+func enableAddon(addOnManager *manager.AddOnManager, addOnName string, priority int) {
+	addOnConfig, err := addOnManager.Enable(addOnName, priority)
 	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Unable to enable add-on %s: %s", addonName, err.Error()))
+		atexit.ExitWithMessage(1, fmt.Sprintf("Unable to enable add-on %s: %s", addOnName, err.Error()))
+	} else {
+		fmt.Println(fmt.Sprintf("Addon %s enabled", addOnName))
```
Comment (budhrg) [inline cmd/minishift/cmd/addon/addon_enable.go]: wrapping in single quote give visibility to name of addon

File: cmd/minishift/cmd/addon/addon_disable.go (hunk 1 above)
Comment (LalatenduMohanty) [inline cmd/minishift/cmd/addon/addon_disable.go]: @budhrg I think `add` `on` is not a single word. Also in other places , variables and functions are named as `addOn`* so I made it consistent.

**Comment by budhrg** [comment]
retest this please

File: cmd/minishift/cmd/addon/addon_disable.go (hunk 2 above)
Comment (budhrg) [inline cmd/minishift/cmd/addon/addon_disable.go]: @LalatenduMohanty  and this one ?

**Pull request merged**